I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 73, in script
    jump d_1_andar
ScriptError: could not find label 'd_1_andar'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 73, in script
    jump d_1_andar
  File "C:\Users\thiag\Documents\RenPy\renpy-8.0.3-sdk\renpy\ast.py", line 1974, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\thiag\Documents\RenPy\renpy-8.0.3-sdk\renpy\script.py", line 927, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'd_1_andar'.

Windows-10-10.0.19041 AMD64
Ren'Py 8.0.3.22090809
OINE_Project_Test 1.0
Mon Jun  5 16:40:40 2023
